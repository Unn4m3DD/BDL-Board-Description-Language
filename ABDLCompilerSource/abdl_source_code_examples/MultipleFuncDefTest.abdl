int f1():
  return 0;
end f1;

int f1():
  return 0;
end f1;

string f2():
  return "";
end f2;

int f2():
  return 1;
end f2;

main:
end main;

main:
end main;

on_move:

end on_move;

on_move:

end on_move;