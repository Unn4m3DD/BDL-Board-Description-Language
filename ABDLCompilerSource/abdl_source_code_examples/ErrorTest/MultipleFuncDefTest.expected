main function declared multiple times(5:14)
on_move function declared multiple times(5:14)
Multiple functions declared with the same name (2:0): f1
Multiple functions declared with the same name (4:0): f2